cli/args/flags.rs

```diff
@@ -532,14 +520,118 @@ pub struct Flags {
   pub no_remote: bool,
   pub no_lock: bool,
   pub no_npm: bool,
-  pub no_prompt: bool,
   pub reload: bool,
   pub seed: Option<u64>,
   pub strace_ops: Option<Vec<String>>,
   pub unstable_config: UnstableConfig,
   pub unsafely_ignore_certificate_errors: Option<Vec<String>>,
   pub v8_flags: Vec<String>,
   pub code_cache_enabled: bool,
+  pub permissions: PermissionFlags,
```

Comment:
That's a nice cleanup but I wish it was done in a separate PR, makes reviewing this one a lot harder.